docs/references/preferences.rst

```diff
@@ -0,0 +1,47 @@
+.. _preferences:
+
+Preferences
+###########
+
+This section describes the ``Prefer`` header value that PostgREST honors.
+
+.. _prefer_handling:
+
+Handling Prefer
+===============
+
+The handling prefer accepts two values ``strict`` and ``lenient``. The ``strict`` value does not allow invalid preferences specified in the ``Prefer`` header. The ``lenient`` value ignore any invalid preferences given in the ``Prefer`` header.
```

Comment:
More details. Right now we don't fail on "unfulfillable" preferences but we might in the future.
```suggestion
The server ignores unrecognized or unfulfillable preferences by default. You can control this behavior with the ``handling`` preference. It can take two values: ``lenient`` (the default) or ``strict``.
```